public/theme/extensions-2021/index.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main %}
+    <main id="content" tabindex="-1">
+        <div class="subheader">
+            {% setcontent homepage = "homepage" limit 1 returnsingle %}
+            {% if homepage %}
+            <h1>{{ homepage.title }}</h1>
+
+            <div class="teaser">{{ homepage.teaser }}</div>
+        </div>
+        {% else %}
+            {# Remove this include if you don't need it anymore. #}
+            {{ include('partials/_no_content.twig') }}
+        {% endif %}
+        <div class="content-wrapper">
+            <h2>Most recently updated Packages</h2>
+            {% setcontent records = 'packages' order '-time_updated' limit 6 %}
+            {% for record in records %}
+                <div class="news-wrapper">
+                    {% include('partials/_sub_listing_package.twig') %}
+                </div>
+            {% else %}
+                <p>{{ __("general.phrase.no-content-found") }}</p>
+            {% endfor %}
+        </div>
+
```

Comment:
Since we don't show all the extensions here, let's add 2 call to action buttons for extensions and themes.

Reply:
ok